geomstats/information_geometry/statistical_metric.py

```diff
@@ -0,0 +1,49 @@
+"""Statistical metric to equip a statistical manifold."""
+
+import geomstats.backend as gs
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class StatisticalMetric(RiemannianMetric):
+    """
+    Defines statistical metric and connection induced from a divergence.
```

Comment:
Respect docstring conventions laid out here: https://geomstats.github.io/contributing/index.html#writing-docstrings

Eg: Imperativ form for verbs --> Define not Defines
Put verb directly after """